src/users/users.service.ts

```diff
@@ -53,11 +59,18 @@ export class UsersService {
     return user;
   }
 
-  async updateUserInfo(email: string, info: UserDto): Promise<User> {
+  async updateUserInfo(
+    request: FastifyRequest,
```

Comment:
It shouldn't depend on `FastifyRequest`, to decode token and obtain claims you should either use middleware or do that in the controller